openapi/openapi.yaml

```
@@ -519,6 +503,68 @@ tags:
     description: |-
       Use this operation to check the status of the Rebilly API.
       No authentication is required.
+  - name: Storefront account
+    description: |-
+      Use these operations to manage Storefront accounts.
+      An account in which you are subscribed to in order to use the Rebilly product.
+      APIs intended to be used directly from an untrusted browser.
+  - name: Storefront billing portals
+    description: |-
+      Use these operations to access Storefront billing portals.
+      APIs intended to be used directly from an untrusted browser.
+  - name: Storefront checkout forms
+    description: |-
+      Use these operations to access Storefront checkout forms.
+      APIs intended to be used directly from an untrusted browser.
```

suggestion: to add more detail
```suggestion
      Use these operations to access storefront checkout forms.
      
      Storefront operations interact directly with the customer, 
      and provide the customer with access to their own data. 
      
      To use storefront operations, the customer must be issued a JWT with limited permissions. For more information, see [Customer authentication](https://www.rebilly.com/catalog/all/Storefront-account/StorefrontPostLogin) and [Exchange an authentication token](https://www.rebilly.com/catalog/all/Customer-authentication/PostAuthenticationTokenExchange).
      Storefront operations are intended to be used directly from an untrusted browser.
```